src/pages/Home.tsx

```diff
@@ -25,41 +25,88 @@ const Home: React.FC = () => {
       </h1>
 
       <div className="card-container">
-        <InfoCard
-          title="PAX booked"
-          value={data?.pax ?? 0}
-          delta={data?.pax_delta ?? 0}
-          currency=""
-        />
-        <InfoCard
-          title="GMV"
-          value={data?.gmv ?? 0}
-          delta={data?.gmv_delta ?? 0}
-          currency={data?.currency ?? 'USD'}
-        />
+        <Row gutter={[16, 16]}>
+          <Col
+            xs={{ span: 24 }}
```

Comment:
You may use shorthand `xs={24}`. Also there is no need to configure each breakpoint as higher breakpoints automatically use values of smaller if they don't have their own. So in this particular case following will do the same as your code:
```
<Col xs={24} sm={12} xl={8}>
...
```